packages/next/src/server/app-render/make-get-server-inserted-html.tsx

```diff
@@ -90,12 +86,9 @@ export function makeGetServerInsertedHTML({
 
```

Comment:
The logic to skip rendering is here ^

Just highlighting in case GH interface collapses it and it's not obvious